destination/src/main/scala/quasar/plugin/sqlserver/destination/TempTableFlow.scala

```diff
@@ -88,14 +90,22 @@ object TempTableFlow {
     } yield {
       val flow = new TempTableFlow {
         def ingest(chunk: Chunk[CharSequence]): ConnectionIO[Unit] =
-          tempTable.ingest(chunk) >> commit
+          retry(tempTable.ingest(chunk)) >>
```

Comment:
Is it safe to retry this without a `rollback`?

Reply:
Isn't it? It's one command `INSERT INTO`, I supposed it's atomic. Anyways, `rollback` before timeout and retry is awesome idea :+1:

Reply:
It is a [batch of commands](https://github.com/precog/quasar-plugin-sql-server/blob/master/destination/src/main/scala/quasar/plugin/sqlserver/destination/package.scala#L166) potentially consisting of many `INSERT INTO` statements, one per element of the chunk.